source/wp-content/themes/wporg-documentation-2022/parts/header-home.html

```diff
@@ -0,0 +1,19 @@
+<!-- wp:wporg/global-header /-->
+
+<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"backgroundColor":"charcoal-2","textColor":"white","className":"is-sticky","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull is-sticky has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color">
+	
+	<!-- wp:group {"align":"full","style":{"border":{"top":{"color":"var:preset|color|opacities-white-10","width":"1px"}},"spacing":{"padding":{"top":"18px","bottom":"18px","left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}},"layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group alignfull" style="border-top-color:var(--wp--preset--color--opacities-white-10);border-top-width:1px;padding-top:18px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:18px;padding-left:var(--wp--preset--spacing--edge-space)">
+		
+		<!-- wp:paragraph {"fontSize":"small"} -->
+		<p class="has-small-font-size">&nbsp;</p>
```

Comment:
Without content here the header collapses on mobile. We had a minimum site set on the breadcrumb container to work around this before. I know this is hacky but it doesn't require adding a fixed CSS value which is brittle because it may not match a font size change.

Reply:
`1.71em` wasn't quite right (ended up calculated at ~27px, but I've used a calculation in https://github.com/WordPress/wporg-documentation-2022/pull/67/commits/9e3dcd64875d1be1757d97b5e716ad149c3eeabf